bundle/bundle.go

```diff
@@ -119,6 +119,16 @@ func (img *InvocationImage) DeepCopy() *InvocationImage {
 	return &img2
 }
 
+// Validate the image contents.
+func (img InvocationImage) Validate() error {
+	switch img.ImageType {
+	case "docker", "oci":
+		return validateDockerish(img.Image)
```

Comment:
Note that as we already vendor `docker/distribution`, we can use https://github.com/docker/distribution/blob/master/reference/reference.go#L189 instead to parse image reference?

Reply:
I'm not sure whether this should necessarily be part of this PR, but I agree we should have more robust validation.

Reply:
I can totally be in a follow up, as `validateDockerish` wasn't introduced in this PR.

Reply:
Good call; filed https://github.com/cnabio/cnab-go/issues/196